snooty.toml

```
@@ -25,6 +25,7 @@ mongosh = "``mongosh``"
 driver = "node"
 driver-long = "MongoDB Node.js driver"
 driver-short = "Node.js driver"
+mdb-server = "MongoDB server"
```

I think server should be capitalized here since MongoDB Server is the official product name

```suggestion
mdb-server = "MongoDB Server"
```